sentry-samples/sentry-samples-spring/src/main/java/io/sentry/samples/spring/WebConfig.java

```diff
@@ -24,7 +24,7 @@
    * @return RestTemplate
    */
   @Bean
-  RestTemplate restTemplate(IHub hub) {
+  RestTemplate restTemplate(final IHub hub) {
```

Comment:
always add @NotNull or @Nullable annotations too when touching code, so little by little, we'd have a better Kotlin interop. with Kotlin null-safety

Reply:
These are samples. As I understand, we did not add these annotation to samples as they are not used by Kotlin code plus adding them makes the samples less approachable to users.

Reply:
yep makes sense, my bad